deno/lib/locales/en.ts

```diff
@@ -71,10 +71,11 @@ const errorMap: ZodErrorMap = (issue, _ctx) => {
         message = `Array must contain ${
           issue.exact ? "exactly" : issue.inclusive ? `at least` : `more than`
         } ${issue.minimum} element(s)`;
-      else if (issue.type === "string")
+      else if (issue.type === "string"){
         message = `String must contain ${
           issue.exact ? "exactly" : issue.inclusive ? `at least` : `over`
-        } ${issue.minimum} character(s)`;
+          } ${issue.minimum} character${issue.minimum>1 ? "(s)" : ""}`;  
```

Comment:
Why are you keeping the `(` and `)` on `(s)` if now you know it must be plural?

Maybe:
```suggestion
          } ${issue.minimum} character${issue.minimum>1 ? "s" : ""}`;  
```
?

Also, in English, `0` quantities should also be pluralized: we say `0 apples`, not `0 apple`.